instrumentation/trilogy/test/opentelemetry/instrumentation/trilogy/instrumentation_test.rb

```diff
@@ -164,6 +164,12 @@
         _(span.attributes[OpenTelemetry::SemanticConventions::Trace::DB_SYSTEM]).must_equal 'mysql'
         _(span.attributes[OpenTelemetry::SemanticConventions::Trace::DB_STATEMENT]).must_equal 'DESELECT ?'
       end
+
+      it 'includes row count' do
+        client.query('SELECT 1')
```

Comment:
I only see `SELECT`s in this test file, so I'm not sure what the best way to test `affected_rows` is.

Reply:
You should be able to run an insert and subsequent update command to assert the behaviour here, no?